customRoutes/publishcve.js

```
@@ -0,0 +1,168 @@
+        var result = await new Promise( res => { publishCve(j.cveMetadata.cveId, isupdate, j.containers.cna, res)})
+        if (!result) {
+            res.json({"body":"Push to cve.org success."});
+            
+            var s2 = email.sendemail({"to":"security@khulnasoft.com",
+                                      "subject":j.cveMetadata.cveId+" was pushed to cve.org",
+                                      "text":"push by "+req.user.username+" success",
+                                     }).then( (x) => {  console.log("sent CVE push mail "+x);});
+        } else {
+            res.json({"body":"Push to cve.org failed. "+result});
+
+            var s2 = email.sendemail({"to":"security@khulnasoft.com",
```

Biome (1.9.4)

[error] Shouldn't redeclare 's2'. Consider to delete it or rename it.

's2' is defined here:


(lint/suspicious/noRedeclare)